### PR TITLE
Judith/releases page

### PR DESCRIPTION
**Pull request by judithpatudith**
## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-website#testing-your-updates-locally) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-website#contribution-workflow).

### CODE REVIEW

**Comment by judithpatudith** [comment]
Hey @sascala and @pleia2 , I had trouble merging the old pull request for the new release page because there were too many merge conflicts for me to figure out, so I put all the final changes on a new branch. Please check this PR out thoroughly and feel free to ping me with questions. I asked for reviews from both of you cause this one has been a doozy. Thanks!

File: src/releases/index.jade [hunk 1]
```diff
@@ -6,87 +6,89 @@ block content
...
+          .release-card.release-stable.col.col-12.md-col-6.md-pr1.relative
+            div.bg-white.center.inner.px3.pt3.pb3.mb2.md-mb0
+              h3.mb0 Latest Stable
+              p.release-version.pb1 DC/OS 1.9.2
+              p.hero-list__item__copy__paragraph.release-copy Latest stable release. Tested and ready for use in production.
+              a.cta.cta--text.release-notes.pb0(href='/releases/1.9.2') Release Notes
 
-        .cards.flex.flex-auto.flex-wrap
+              div.release-buttons.pb3
+                a.cta.cta--button.inline-block(href='https://downloads.dcos.io/dcos/stable/aws.html') Amazon
```
Comment (sascala) [inline src/releases/index.jade]: ^ I think you want to add `, target='blank')` to the Amazon link

File: src/releases/index.jade [hunk 2]
```diff
@@ -6,87 +6,89 @@ block content
...
+            strong.block.mb2 #{ release.month } #{ release.day }, #{ release.year }
+            if(release.description)
+              p.my1.mb3 #{ release.description }
+            strong.block.mb1 Features
+            if(release.features)
+              ul.mb3
+                each feature in release.features
+                  li #{ feature }
 
+            h3 Get the latest relese of #{ release.title }
```
Comment (joerg84) [inline src/releases/index.jade]: release

File: src/releases.json [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      "Jobs.",
+      "Networking improvements.",
+      "Binary services CLIs.",
+      "Offline Universe (package management service).",
+      "Non-root user config for data services.",
+      "Apache Mesos 1.0.3.",
+      "Marathon 1.3.9 (native in the DC/OS UI)."
+    ],
+    "downloadlink": {
+      "amazon": "https://downloads.dcos.io/dcos/stable/aws.html",
```
Comment (joerg84) [inline src/releases.json]: This shouldn't point to stable (being 1.9)

File: src/releases.json [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      "azure": "https://downloads.dcos.io/dcos/stable/azure.html",
```
Comment (joerg84) [inline src/releases.json]: dito

**Pull request merged**